add PR 566 in changelog (#569)
XanaduAI/pennylane: .github/CHANGELOG.md

<h3>Improvements</h3>

> * Made the input check functions in :mod:`pennylane.templates.utils` public,
>   so they can be used when creating new templates, and they are visible in the
>   API documentation.

Reviewer: ```suggestion
* The input check functions in :mod:`pennylane.templates.utils` are now public
  and visible in the API documentation.
```
